cost-analyzer/templates/_helpers.tpl

```diff
@@ -1335,7 +1335,7 @@ Begin Kubecost 2.0 templates
   {{- end }}
   args: ["cloud-cost"]
   ports:
-    - name: tcp-api
+    - name: cloud-cost
```

Comment:
Just curious why you're doing this? Not a big deal! I don't think the name of the port matters.

Reply:
I should have added more detail for that.

in single pod installs:

```
helm install kubecost284 -n kubecost284 kubecost2x/cost-analyzer --version 2.8.4 --create-namespace
```

I1031 15:15:05.103453   50753 warnings.go:110] "Warning: spec.template.spec.containers[3].ports[0]: duplicate port name \"tcp-api\" with spec.template.spec.containers[2].ports[0], services and probes that select ports by name will use spec.template.spec.containers[2].ports[0]"